pkg/templates/nstemplatetiers/nstemplatetier_generator_whitebox_test.go

```diff
@@ -410,10 +410,22 @@ func assertNamespaceTemplate(t *testing.T, decoder runtime.Decoder, actual templ
 	require.NoError(t, err)
 	assert.Equal(t, expected, actual)
 	// Assert expected objects in the template
-	// Each template should have one Namespace, one RoleBinding, one LimitRange and three NetworkPolicy objects
+	// Each template should have one Namespace, one RoleBinding, one LimitRange and a varying number of NetworkPolicy objects depending on the namespace kind
+
+	// Template objects count
+	if tier == "team" {
```

Comment:
Let's use switch/case instead.

Reply:
Done in 95ca667b15c6c242c200cfb797b9eecb577d188c